### DISCUSSION TITLE
fitBounds() to lng/lat + radius

### DISCUSSION THREAD

**Discussion by MarkusWendorf** [maplibre/maplibre-gl-js#4426] [Q&A] [answered]
Hello there,

I want to use the `fitBounds` function to zoom the map to a specific location. However, I only have a position (lat/lng) and a radius around the position.

Is there a way to use `fitBounds` with lng/lat + radius? 

I was thinking about using something like this to convert it to a polygon first: https://github.com/gabzim/circle-to-polygon/blob/master/index.js

**Comment by HarelM** [comment, marked as answer]
You'd need to calculate the bounding box of the circle and feed that into the fit bounds function.
It should be fairly trivial math I believe.
